packages/plugins/model-define/bayesian-model-define/src/script.ts

```diff
@@ -71,18 +72,16 @@ export const processPredictData = function (data: any, all_words_list_path: stri
     return features;
   };
 
-  return MakeWordsSet(stopwords_file).then((stopwords_set) => {
-    const feature_words = words_dict(all_words_list, stopwords_set);
-    return word_cut.map((text: string) => {
-      return text_features(text, feature_words);
-    });
-  });
+  const stopwords_set = await MakeWordsSet(stopwords_file);
+  const feature_words = words_dict(all_words_list, stopwords_set);
+  const feature_list = [ text_features(word_cut, feature_words) ];
+  return feature_list;
```

Comment:
Could it return [ text_features(word_cut, feature_words) ]; directly？

Reply:
done